snap-template/snapcraft.yaml

```diff
@@ -0,0 +1,30 @@
+name: my-snap  # the name of the snap
+version: 0  # the version of the snap
+summary: This is my-snap's summary  # 79 char long summary
+description: |
```

Comment:
Why is it only a `|`? I've always put something on this line, but I could just not know what I'm talking about ;)